examples/bank/bank.go

```diff
@@ -93,18 +96,84 @@ func (bank *Bank) moveMoney(from, to []byte, amount int64) bool {
 			return errRead
 		}
 		// Update both accounts.
-		txn.Prepare(client.Put(proto.Key(from), []byte(fmt.Sprintf("%d", fromValue-amount))))
-		txn.Prepare(client.Put(proto.Key(to), []byte(fmt.Sprintf("%d", toValue+amount))))
+		txn.Prepare(client.Put(proto.Key(from), []byte(fmt.Sprintf("%d", fromValue-amount))),
```

Comment:
On Sat, May 2, 2015 at 4:29 PM Tobias Schottdorf <notifications@github.com>
wrote:

> In examples/bank/bank.go
> <https://github.com/cockroachdb/cockroach/pull/881#discussion_r29550415>:
>
> > @@ -93,18 +96,84 @@ func (bank *Bank) moveMoney(from, to []byte, amount int64) bool {
> >  			return errRead
> >  		}
> >  		// Update both accounts.
> > -		txn.Prepare(client.Put(proto.Key(from), []byte(fmt.Sprintf("%d", fromValue-amount))))
> > -		txn.Prepare(client.Put(proto.Key(to), []byte(fmt.Sprintf("%d", toValue+amount))))
> > +		txn.Prepare(client.Put(proto.Key(from), []byte(fmt.Sprintf("%d", fromValue-amount))),
>
> I think I asked before but I don't remember if I got an answer: You're
> deliberately not using Increment and our Integer types, right?
>
Yeah I'm deliberately not using increment. I suppose we might want to hold
more than an amount in an account. Thanks for reviewing the PR.



> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/881/files#r29550415>.
>